### PR TITLE
lame: fix wrap and add fixes from other PR

### PR DESCRIPTION
**Pull request by neheb**
Added https://github.com/mesonbuild/wrapdb/pull/791 and actually fix wrap to work as a subproject.

ping @amyspark

### CODE REVIEW

**Comment by neheb** [comment]
Funny. frontend binaries compile now.

File: releases.json [hunk 1]
```diff
@@ -887,6 +887,7 @@
       "mpglib"
     ],
     "versions": [
+      "3.100-6",
```
Comment (amyspark) [inline releases.json:890]: You'll need to reorder this one to the front (and maybe co-authored-by it?).

File: subprojects/packagefiles/lame/meson_options.txt [hunk 2]
```diff
@@ -1,4 +1,6 @@
-option('decoder', type : 'boolean', value : false,
-       description : 'Build the decoder')
-option('tools', type : 'boolean', value : false,
+option('decoder', type : 'boolean', value : true,
+       description : 'Build the mpg123 decoder')
+option('iconv', type : 'feature',
+       description : 'Build tools with iconv support')
+option('tools', type : 'feature',
```
Comment (amyspark) [inline subprojects/packagefiles/lame/meson_options.txt:5]: Is the default value `auto` here? If so, you could make `get_option('decoder').disable_auto_if(not curses_dep.found())` and then use the returned value with `decoder.allowed()` for clarity.

**Review by amyspark — APPROVED** [review]
Just those two nitpicks, otherwise LGTM. Closing mine.

File: releases.json (hunk 1 above)
Comment (neheb) [reply releases.json:890]: Not sure what you mean.

File: subprojects/packagefiles/lame/meson_options.txt (hunk 2 above)
Comment (neheb) [reply subprojects/packagefiles/lame/meson_options.txt:5]: feature options default to auto yes. Not sure what decoder has to do with tools. It probably has to remain enabled as there is a dependency listed in the wrap file.

File: releases.json (hunk 1 above)
Comment (amyspark) [reply releases.json:890]: After the version bump come 10 commits introducing further changes. These should all come *before* the bump?

imagen

Comment (neheb) [reply releases.json:890]: Fixed.

**Pull request merged**